buildcontainer/Makefile

```diff
@@ -1,14 +1,18 @@
 include .env
 
 REGISTRY           ?= ghcr.io
-TAG_VERSION        ?= $(shell git describe --tags --abbrev=0)
+TAG_VERSION        ?= snapshot
+OWNER              ?= goreleaser
+GIT_REPO           ?= https://github.com/goreleaser/goreleaser-cross
 
 ifeq ($(REGISTRY),)
-	IMAGE_BASE_NAME := goreleaser/goreleaser-cross-base:$(TAG_VERSION)
-	IMAGE_NAME      := goreleaser/goreleaser-cross:$(TAG_VERSION)
+	IMAGE_BASE_NAME  := $(OWNER)/goreleaser-cross-base:$(TAG_VERSION)
+	IMAGE_NAME       := $(OWNER)/goreleaser-cross:$(TAG_VERSION)
+	TURBO_IMAGE_NAME := $(OWNER)/turbo-cross:$(TAG_VERSION)
 else
-	IMAGE_BASE_NAME := $(REGISTRY)/goreleaser/goreleaser-cross-base:$(TAG_VERSION)
-	IMAGE_NAME      := $(REGISTRY)/goreleaser/goreleaser-cross:$(TAG_VERSION)
+	IMAGE_BASE_NAME  := $(REGISTRY)/$(OWNER)/goreleaser-cross-base:$(TAG_VERSION)
+	IMAGE_NAME       := $(REGISTRY)/$(OWNER)/goreleaser-cross:$(TAG_VERSION)
+	TURBO_IMAGE_NAME := $(REGISTRY)/$(OWNER)/turbo-cross:$(TAG_VERSION)
 endif
 
 OSX_SDK            := MacOSX12.0.sdk
```

Comment:
Still a bunch of macOS stuff hanging around.

Reply:
Removed